src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -163,7 +163,7 @@ namespace AppInstaller::CLI::Workflow
 
     void ReadImportFile(Execution::Context& context)
     {
-        std::ifstream importFile{ context.Args.GetArg(Execution::Args::Type::ImportFile) };
+        std::ifstream importFile(std::string(context.Args.GetArg(Execution::Args::Type::ImportFile)));

```

Comment:
I made that change, and am happy to do the test data files too, but since the ifstreams in the tests are being constructed with `std::filesystem::path` do they still need to be converted to UTF16?

Reply:
Ok, it looks like there would only be one place that is likely to need updating, and that is really only used by the tests on the build agent.  The default scenario should work already.  No need to bother with it unless an issue actually arises.